### PR TITLE
aia buttons

### PR DESCRIPTION
**Pull request by JefMari**
capture1
capture

### CODE REVIEW

**Comment by JefMari** [comment]
@srph

File: styles/components/button.scss [hunk 1]
```diff
@@ -45,8 +45,18 @@
...
   &.-sun { @include btn-variant($brand-sun); }
+  &.-crimson { @include btn-variant($brand-crimson); }
+  &.-clear {
+    @include btn-variant(transparent, $brand-primary);
+
+    &:hover,
+    &:focus {
+      color: darken($brand-primary, 5%);
+      text-decoration: underline;
+      background: transparet;
```
Comment (srph) [inline styles/components/button.scss:57]: `transparet`?

File: styles/components/button.scss [hunk 2]
```diff
@@ -68,6 +78,7 @@
     width: 100%;
     margin-right: auto !important;
     margin-left: auto!important;
```
Comment (srph) [inline styles/components/button.scss:80]: Remove `!important` rule for both margins

File: docs/sections/Docs-Buttons/index.jsx [hunk 3]
```diff
@@ -58,6 +60,7 @@ export default class Buttons extends React.Component {
             <button className="btn -emerald -rounded">Button</button>
             <button className="btn -sun -rounded">Button</button>
             <button className="btn -crimson -rounded">Button</button>
+            <button className="btn -clear -rounded">Button</button>
```
Comment (srph) [inline docs/sections/Docs-Buttons/index.jsx:63]: Huh?

**Comment by srph** [comment]
- I think you should name it as `-clear`.
- 1 PR = 1 issue to address, pleaaase.
- Descriptive PR titles next time, please.
- Reference the issue next time by copy-pasting the issue link (https://github.com/dev-onenetworkecommerce/aia/issues/80)
- Screenshot for https://github.com/dev-onenetworkecommerce/aia/issues/80?

**Comment by srph** [comment]
https://github.com/dev-onenetworkecommerce/aia/pull/87

**Pull request closed without merging**